gauge.go

```diff
@@ -103,6 +107,9 @@ func (g *Gauge) Buffer() []Point {
 				p.Bg |= AttrReverse
 			}
 
+			if g.PercentColorHighlighted != Attribute(uint16max) {
```

Comment:
I think define an attribute like `var ColorClear = Attribute(0xFFFF)` and add some docs on it might be better.